### PR TITLE
INC-993 Remove IEP endpoints and generation on receive and transfer

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java [hunk 1]
```diff
@@ -240,7 +244,7 @@ private InmateDetail getOffenderDetails(final InmateDetail inmate, final boolean
             }
             if (extraInfo) {
                 inmate.setAliases(repository.findInmateAliases(bookingId, "createDate", Order.ASC, 0, 100).getItems());
-                inmate.setPrivilegeSummary(bookingService.getBookingIEPSummary(bookingId, false));
```
Comment (Mjwillis) [inline src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:243]: Like to remove this code next

Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:243]: Don't think anyone looks at IEP through this endpoint but need to check

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java [hunk 2]
```diff
@@ -240,7 +244,7 @@ private InmateDetail getOffenderDetails(final InmateDetail inmate, final boolean
...
+                inmate.setPrivilegeSummary(getBookingIEPSummary(bookingId));
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:247]: do you know who uses this?

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java (hunk 2 above)
Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:247]: Don't think anyone does

**Pull request merged**